### PR TITLE
Feature/remove fixed width

### PR DESCRIPTION
**Pull request by DImitarTab**

### CODE REVIEW

File: source/about.html [hunk 1]
```diff
@@ -1,51 +1,52 @@
...
-                    world film releases on the week they air in Bulgaria.
-                </p>
-                <div class="images">
-                    <img src="images/seats.jpg" class="interior" alt="Cinema seats">
-                    <img src="images/screen.jpg" class="interior" alt="Cinema screen">
-                </div>
-
-            </div>
-        </section>
+      <section class="header">
```
Comment (emiltsokov) [inline source/about.html]: why not using header tag here

File: source/about.html [hunk 2]
```diff
@@ -1,51 +1,52 @@
...
+        <div class="header-name"><h1>Sliven Cinema</h1></div>
+      </section>
+      <section class="main-body">
+        <div class="nav">
```
Comment (emiltsokov) [inline source/about.html]: why not use nav tag here

File: source/index.js [hunk 3]
```diff
@@ -14,8 +14,29 @@ $(".seats").click(function() {
   });
   $(".form-seats").val(selectedSeats.join(", "));
   $(this).toggleClass("clicked");
+  $(this)
+    .parent()
+    .toggleClass("selected");
 });
 
+$(document).ready(function() {
```
Comment (emiltsokov) [inline source/index.js]: Please use the ' the recommended way of calling' .ready()
https://api.jquery.com/ready/

File: source/index.js [hunk 4]
```diff
@@ -14,8 +14,29 @@ $(".seats").click(function() {
   });
```
Comment (emiltsokov) [inline source/index.js]: $(".seats").click() - move this inside document ready function

File: source/index.js [hunk 5]
```diff
@@ -14,8 +14,29 @@ $(".seats").click(function() {
...
 });
 
+$(document).ready(function() {
+  var url = new URL(document.location);
+  var params = url.searchParams;
+  var paper = params.get("movie");
+  $(".movie-select").val(paper);
+});
+
+const urlParams = new URLSearchParams(window.location.search);
```
Comment (emiltsokov) [inline source/index.js]: cont declaration should be at the top of the file 
prefer useing let or const not var (e.g var selectedSeats)

File: source/style/main/main.scss [hunk 6]
```diff
@@ -58,7 +58,7 @@ h2 {
   display: flex;
   flex-direction: column;
   justify-content: center;
-  min-width: 500px;
+  // min-width: 500px;
   flex: 0 2 1720px;
```
Comment (emiltsokov) [inline source/style/main/main.scss]: what is this about 1720px

File: source/about.html (hunk 1 above)
Comment (DImitarTab) [reply source/about.html]: header tag added

File: source/about.html (hunk 2 above)
Comment (DImitarTab) [reply source/about.html]: nav tag added

File: source/index.js (hunk 3 above)
Comment (DImitarTab) [reply source/index.js]: added the recommended way of calling .ready()

File: source/index.js (hunk 4 above)
Comment (DImitarTab) [reply source/index.js]: function is now inside the document .ready()

File: source/index.js (hunk 5 above)
Comment (DImitarTab) [reply source/index.js]: put all variables and const at the top and changed var to let

File: source/style/main/main.scss (hunk 6 above)
Comment (DImitarTab) [reply source/style/main/main.scss]: It is a left out from when i tested flex short-hand property and wanted to an element to take the width of its containet. Changed it to width:100%

File: source/index.js [hunk 7]
```diff
@@ -4,18 +4,59 @@ import "./style/style.scss";
...
+    window.history.pushState(
+      {},
+      "Hide",
+      "http://localhost:8008/book-ticket.html"
+    );
+  }
 });
 
+window.onload = function() {
+  SetDefaultValue();
```
Comment (emiltsokov) [inline source/index.js:53]: this one also can go inside the document ready

Comment (emiltsokov) [reply source/index.js:53]: will leave this js refactoring for some following PR

**Pull request merged**